server/chat-commands/info.ts

```diff
@@ -2509,14 +2509,14 @@ export const commands: ChatCommands = {
 				return this.errorReply('Invalid image');
 			}
 		}
-		buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray"><small>(Requested by ${request.name})</small>`;
+		buf += Utils.html`<br /><div class="infobox"><small>(Requested by ${request.name})</small>`;
 		if (request.comment) {
-			buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray">${request.comment}</p>`;
+			buf += Utils.html`<br />${request.comment}</small></div>`;
 		} else {
-			buf += `</small></p>`;
+			buf += `</small></div>>`;
 		}
-		this.addBox(buf);
-		room.update();
+		room.add(`|c|${request.name}|/uhtml ${request.name},${buf}`).update();
+		this.privateModAction(`${user.name} has approved the display of ${request.name}'s media approval.`);
```

Comment:
```suggestion
		this.privateModAction(`${user.name} approved showing ${request.name}'s media.`);
```

Reply:
I intentionally made this longer so /approvallog would scrape them easier.

Reply:
You shouldn't add bad grammar to give approvallog an easier time (not to mention that shorter search terms should have better perf). Just add an actual token to search for if it's an issue.

Reply:
```suggestion
		this.privateModAction(`${user.name} approved showing media from ${request.name}.`);
```

Reply:
.